build-farm/make-adopt-build-farm.sh

```diff
@@ -18,6 +18,17 @@ set -e
 
 PLATFORM_SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
+## Sanity check as this script requires various environment variables so fail
+## fast if one or more are not present. MAYBE autodetect defaults in future?
+
+SANEVARS=0
+[ -z "$JAVA_TO_BUILD" ] && echo JAVA_TO_BUILD not defined - set to e.g. jdk8u && SANEVARS=1
+[ -z "$TARGET_OS"     ] && echo TARGET_OS not defined - set to e.g. linux     && SANEVARS=1
+[ -z "$VARIANT"       ] && echo VARIANT not defined - set to e.g. hotspot     && SANEVARS=1
+[ -z "$ARCHITECTURE"  ] && echo ARCHITECTURE not defined - set to e.g. x64    && SANEVARS=1
+[ -z "$FILENAME"      = && echo FILENAME not defined - set to e.g. jdk.tar.gz && SANEVARS=1
```

Comment:
this line is not like the others...

Reply:
Not entirely sure how I managed that since I did run it to test ... Must have hit something random on the keyboard afterwards. Fixed.

Reply:
Made a hash of something else though - give me 5 minutes to clean up

Reply:
Sorted